driver/src/main/java/com/code_intelligence/jazzer/autofuzz/Meta.java

```diff
@@ -445,27 +454,31 @@ static Object consume(FuzzedDataProvider data, Type genericType, AutofuzzCodegen
       if (visitor != null) {
         throw new AutofuzzError("codegen has not been implemented for Method.class");
       }
-      return data.pickValue(sortExecutables(YourAverageJavaClass.class.getMethods()));
+      return data.pickValue(getAccessibleMethods(YourAverageJavaClass.class));
     } else if (type == Constructor.class) {
       if (visitor != null) {
         throw new AutofuzzError("codegen has not been implemented for Constructor.class");
       }
-      return data.pickValue(sortExecutables(YourAverageJavaClass.class.getConstructors()));
+      return data.pickValue(getAccessibleMethods(YourAverageJavaClass.class));
     } else if (type.isInterface() || Modifier.isAbstract(type.getModifiers())) {
       List<Class<?>> implementingClasses = implementingClassesCache.get(type);
       if (implementingClasses == null) {
-        ClassGraph classGraph =
-            new ClassGraph().enableClassInfo().enableInterClassDependencies().rejectPackages(
-                "jaz.*");
+        ClassGraph classGraph = new ClassGraph()
+                                    .enableClassInfo()
+                                    .ignoreClassVisibility()
+                                    .ignoreMethodVisibility()
+                                    .enableInterClassDependencies()
+                                    .rejectPackages("jaz.*");
         if (!IS_TEST) {
           classGraph.rejectPackages("com.code_intelligence.jazzer.*");
         }
         try (ScanResult result = classGraph.scan()) {
           ClassInfoList children =
               type.isInterface() ? result.getClassesImplementing(type) : result.getSubclasses(type);
-          implementingClasses = children.getStandardClasses()
-                                    .filter(cls -> cls.isPublic() && !cls.isAbstract())
-                                    .loadClasses();
+          implementingClasses =
+              children.getStandardClasses()
+                  .filter(info -> isAccessible(info.getPackageName(), info.getModifiers()))
```

Comment:
`isAccessible` does not check `isAbstract` but `children` could still include `abstract` classes, couldn't it?

Reply:
True, added another `filter` call.